### PR TITLE
To Do list: list structure

### PR DESCRIPTION
**Pull request by oovillagran**
In this project, I built a simple HTML list of To Do tasks(✅). The list was styled according to the specifications listed in this project requirement. This simple web page was built using webpack and served by a webpack dev server.

**Project Requirements**

- Set up a new project with webpack that is based on the webpack exercise previously completed.
- Create an **index.html** file and write the HTML markup here. Create an empty **To Do List** placeholder ([div] or [ul] element). The **index.html** file must be set as a template using the *HTML Webpack Plugin*.
- Create an **index.js** file and set an array of some simple to do tasks (array of objects). Each task object should contain three keys:
     - description [string].
     - completed [bool].
     - index: [number].
- Write a function to iterate over the tasks array and populate an HTML list item element for each task.
- On page load render the dynamically created list of tasks in the dedicated placeholder. The list should appear in order of the index values for each task.
- Create a **style.css** and set rules for the **To Do List**. CSS must be loaded by *Webpack Style/CSS Loader*. 
- All the source files (**index.html**, **index.js**, and **style.css**) must be located in */src* directory and the distribution files will be generated by webpack and served by *webpack dev server* from */dist* folder.

### CODE REVIEW

File: webpack-demo/src/index.js [hunk 1]
```diff
@@ -1,23 +1,58 @@
...
 
-function component() {
-  const element = document.createElement('div');
-  const btn = document.createElement('button');
+class Activity {
+  constructor(description) {
+    this.index = 0;
+    this.completed = false;
+    this.description = description;
+  }
```
Comment (iambenkis) [inline webpack-demo/src/index.js]: - [x] Good job creating the Activity class for creating tasks 👏 , but as required you should create a array that contains all the tasks to be rendered, note that the method you use is another way to do that, but to use a array will give more dynamism to your code 😄 , what do you think of the code below 👍 
```bash
const tasks = [
  {
    description: 'An example description in string',
    completed: false,
    index: 10,
  },
];
```

**Review by iambenkis — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES :recycle::recycle: 

Hi @oovillagran 
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there! 

almost-there

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Follow GitHub flow :heavy_check_mark: 

## Required changes :recycle: :recycle: 

- [x] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me @iambenkis  when you comment, so I can receive the notification or use Slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: webpack-demo/src/index.js (hunk 1 above)
Comment (oovillagran) [reply webpack-demo/src/index.js]: Thanks @iambenkis, I appreciate your kind suggestion!.

**Review by zainsadaqat — APPROVED** [review]
# Pull Request Approved ✅ 🎊 🎉 
## Hello @oovillagran 👋 

## Things to Appreciate 
- [x] Correct Flow ✅ 
- [x] Clean Code 👍 
- [x] No Linter Errors 🔥 
- [x] Descriptive Pull Request and Summary 👌 

### Your project is complete!  There is nothing else to say other than it's time to merge it.

# Congratulations!!! 🎉 🥳 👏 🙌 🎊 ☑️ 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by oovillagran** [comment]
Thanks @zainsadaqat !!!👌

**Pull request merged**